firmware/rhizome.ino

```diff
@@ -80,5 +90,13 @@ void loop() {
  * Delegation function that allows us to call the managed method for publishing periodic updates.
  */
 void doPeriodicUpdates() {
-    sproutList.publishPeriodicUpdates();
+	//Do not attempt to publish updates if disconnected from the cloud
+    if(Particle.connected() == false){
+		if(EEPROM.read(1) == 0x01){
```

Comment:
This is fine for now, but it would be great if we could store the `0x01` somewhere (as a const variable, naturally) so this is easier to understand. I don't know if we would put it in the menu object or some sort of simple settings object, but I could see some place for finding global constants like that being useful.
Just throwing that out there so we start thinking about it.